java-squid/src/main/java/org/sonar/java/model/InternalSyntaxToken.java

```diff
@@ -119,7 +123,7 @@ public boolean isEOF(){
   }
 
   public static InternalSyntaxToken create(AstNode astNode) {
-    Preconditions.checkArgument(astNode.hasToken(), "has no token");
+//    Preconditions.checkArgument(astNode.hasToken(), "has no token");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)